pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -0,0 +1,374 @@
+---
+title: "Professional Development Workshop Survey"
+---
+:ruby
+  dont_cache
+  enrollment_code = request.splat_path_info[1..-1]
+  enrollment = DASHBOARD_DB[:pd_enrollments].where(code: enrollment_code).first
+  pass unless enrollment
+
+  workshop = DASHBOARD_DB[:pd_workshops].where(id: enrollment[:pd_workshop_id]).first
+  pass unless workshop
+
+  plp = DASHBOARD_DB[:professional_learning_partners].where(contact_id: workshop[:organizer_id]).first
+  # Have they ever taken any pd-workshop-survey?
+  is_first_survey = DB[:forms].where(kind: 'PdWorkshopSurvey').count == 0
+
+  # Have they take this particular survey?
+  previous_response = DB[:forms].where(kind: 'PdWorkshopSurvey', source_id: enrollment[:id]).first
+
+  course = workshop[:course]
+  subject = workshop[:subject]
+
+  def local_view(name, *params)
+    view "pd_workshop_survey/#{name.to_s}", *params
+  end
+
+%script{type: "text/javascript", src: "/js/sifter.min.js"}
+%script{type: "text/javascript", src: "/js/microplugin.min.js"}
+%script{type: "text/javascript", src: "/js/selectize.min.js"}
+%script{type: "text/javascript", src: "/js/selectize-fast-click.js"}
+%link{rel: "stylesheet", type: "text/css", href: "/css/selectize.bootstrap3.css"}/
+
+- if previous_response
+  %h1 Thank you.
```

Comment:
Just a small design nit, remove the period in the title and make the text h3?

`%h1 Thank you`
`%h3` You have already completed this survey.`